source/Octopus.Manager.Tentacle/TentacleConfiguration/SetupWizard/ReviewAndRunScriptTabViewModel.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using Autofac;
+using FluentValidation;
+using Octopus.Diagnostics;
+using Octopus.Manager.Tentacle.Infrastructure;
+using Octopus.Manager.Tentacle.Util;
+using Octopus.Tentacle.Util;
+
+namespace Octopus.Manager.Tentacle.TentacleConfiguration.SetupWizard
+{
+    public class ReviewAndRunScriptTabViewModel : ViewModel, IScriptableViewModel
+    {
+        readonly ICommandLineRunner commandLineRunner;
+        readonly IScriptableViewModel wizardModel;
+        TextBoxLogger logger;
+
+        public ReviewAndRunScriptTabViewModel(IScriptableViewModel wizardModel, IComponentContext container)
+        {
+            this.wizardModel = wizardModel;
+            InstanceName = wizardModel.InstanceName;
+            Executable = CommandLine.PathToTentacleExe();
+            Validator = CreateValidator();
+            commandLineRunner = container.Resolve<ICommandLineRunner>();
```

Comment:
As this is the only usage of `IComponentContext` in this class (nice!), please depend on `ICommandLineRunner` directly and remove the usage of the container.